features/tracker-allowlist.json

```diff
@@ -2930,6 +2930,11 @@
                         "domains": ["<all>"],
                         "reason": "https://github.com/duckduckgo/privacy-configuration/issues/366"
                     },
+                    {
+                        "rule": "runtime.strm.yandex.ru/player/video/",
+                        "domains": ["yandex.cloud"],
+                        "reason": "https://github.com/duckduckgo/privacy-configuration/pull/2481"
```

Comment:
I think we switched decisions on this btw. @GuiltyDolphin could confirm but I think we now just put reasons in the text again to avoid the PR dance :)